api/http/handler/webhooks/webhook_execute.go

```diff
@@ -38,15 +38,15 @@ func (handler *Handler) webhookExecute(w http.ResponseWriter, r *http.Request) *
 
 	dockerClient, err := handler.DockerClientFactory.CreateClient(endpoint)
```

Comment:
We should create a switch on the webhook type here, and execute a specific function accordingly.

Please create a new function in this file for the service update webhook.